### DISCUSSION TITLE
Изменения в list-general.txt

### DISCUSSION THREAD

**Discussion by Beast968** [Flowseal/zapret-discord-youtube#285] [Ideas]
А вы думали что можно добавить другие сайты и спокойно заходить на них.
К примеру я добавил instagram.com и спокойно могу просматривать контент без VPN.

Предлагаю добавлять сайты которые нужно добавлять.

Я бы добавил
updates.discord.com
```
*updates.discord.com
rutracker.org
rutracker.com
instagram.com
*instagram.com
pornhub.com
*pornhub.com
```

**Comment by Flowseal** [comment]
Мой репозиторий - это база, с которой можно быстро стартовать, дальше пользователь сам, если захочет, добавит свои хосты.
К слову, не надо так писать хосты.

discord.com - уже учтёт все поддомены (media.discord.com и т.д)
*discord.com - вообще не сработает

**Reply by SkipTyler282** [reply]
Привет! Спасибо за прогу, и дискорд и ютуб работают (пока у себя в провинции). Московский МТС ещё предостоит проверить. Насчёт добавления нового, не подскажешь, пожалуйста, что нужно для добавления протонмейл? Добавил адреса в этот файлик:
proton.me
protonmail.com
protonvpn.com
Вроде грузятся страницы, всё ок. НО, пытаюсь зайти в почту, и бац, браузер (librewolf) начинает ругаться, не пускает, на сертификат ругается. Захожу в почтовый ящик через коммерческий blanc vpn через этот же браузер, всё ок, ничего не ругается ни на какой сертификат. В чём подвох?

**Reply by zredicon** [reply]
![Снимок экрана 2024-10-17 175250](https://github.com/user-attachments/assets/3ae5a71e-ca25-48b6-9c34-ed6ca2980fa7)
что это такое?

**Reply by Eptiy** [reply]
> что это такое?

https://github.com/Flowseal/zapret-discord-youtube/issues/159

https://github.com/ValdikSS/GoodbyeDPI/issues/376#issuecomment-2251796706

**Comment by Eptiy** [comment]
Вот [здесь](https://github.com/1andrevich/Re-filter-lists) есть отфильтрованный список заблокированных ресурсов. Он ещё дорабатывается, но там уже работает большинство сервисов и нет мусора, вроде бесконечных зеркал с казино.

UPD: с ним не работает ютуб, приходится добавлять домены из list-general.

**Reply by madd0ge** [reply]
Как добавить туда твиттер? Какие бы адреса не добавлял всё равно не открывается

**Reply by Eptiy** [reply]
Твиттер там уже есть, дело скорее всего не в списке, а в настройках обхода. Если не открывается на этом билде запрета, можно попробовать [этот](https://github.com/N3M1X10/zapret_build). У меня лично он открывается на обоих. Может не открываться ни на одном, тогда настройки нужно подбирать вручную.

**Reply by madd0ge** [reply]
Блин, и этот не работает с твиттером. Все работает что добавляю кроме самого твиттера. Правда когда запускаю батник, там написано что OpenService failed. Не знаю что это значит..

**Reply by Eptiy** [reply]
> OpenService failed

Так и должно быть, насколько я понимаю, так что можно не обращать внимания.

Можно собрать все домены твиттера и подобрать вручную обход к ним. Как это сделать написано [тут](https://github.com/bol-van/zapret/blob/master/docs/quick_start_windows.txt). Задача не из простых, но только так оно делается.

**Reply by madd0ge** [reply]
Опа, заработало! Спасибо

**Reply by Eptiy** [reply]
Заработало само по себе, или после каких-то действий?

**Reply by madd0ge** [reply]
Да я просто по ссылке перешел, скачал и сделал как там и с первого же шага у меня заработало все что было заблокировано

**Reply by Beast968** [reply]
Это блин качать, и ставить ещё. Пусть один кто то сделает и закинет.

**Reply by Eptiy** [reply]
> Это блин качать, и ставить ещё. Пусть один кто то сделает и закинет.

Разрабы билдов этим и занимаются. Просто универсального решения с обходами нет, у одних работает одно, у других - другое.

**Reply by anadanibunny-crypto** [reply]
titanium.roblox.com
edge-term4.roblox.com
edge-term4-fra4.roblox.com
games.roblox.com

**Comment by Anatomis** [comment]
недавно заблокировали 7tv попытался добавить их домены несработало есть идеи?
![chrome_xi6trYDhKi](https://github.com/user-attachments/assets/461bf76c-9090-4c77-b20e-6dc8a3811e5b)
![Notepad_h4vetdz5ZN](https://github.com/user-attachments/assets/bb494571-cba0-4c0e-be68-abfd6a72f8e5)

**Reply by Flowseal** [reply]
у CDN 7tv есть определённые приколы, пока что почти никак

**Reply by Anatomis** [reply]
ок

**Comment by zredicon** [comment]
![Снимок экрана 2024-10-17 175250](https://github.com/user-attachments/assets/7afc71f6-ad53-45af-bb1f-63d7af4cac88)
что это?

**Comment by dniweINC** [comment]
Ютуб и дс идеально работают, но заношу в list-general другие сайты и они вообще не запускаются. Пробовал рутор, инсту, твиттер, чатгпт и никак. Может что-то не так делаю?

**Reply by Eptiy** [reply]
Инста заблочена не только по доменам, но и по ip, для того, чтобы обойти блок, нужно использовать дополнительные средства.

ChatGPT заблокировал нас извне по российским ip, поможет только смена местоположения.

Твиттер и рутор у меня лично работают, скорее всего ты добавил не все заблокированные домены в список. Но даже если добавил все - настройки обхода могут не подойти. В таком случае попробуй другие билды или настрой обход вручную.

**Reply by Eptiy** [reply]
Инсту можно попробовать разблочить [этим](https://gist.github.com/illvart/6795f384a71157181b06b5c4a421e768#the-hosts-file-location-for) способом.

**Comment by dmitchill** [comment]
этот сервис как то можно использовать еще и для портов ?

**Comment by L1nkooo** [comment]
Можно обойти блокировку envato ?

**Comment by lliinnii** [comment]
Версия от 9 октября при запуске дискор+ютюб попутно анлокала рутрекер (при этом ни в list-discord.txt ни в list-general.txt записи о рутрекере не было).
Сейчас если добавить rutracker.org в list-general.txt и если добавить  172.67.182.196 в ipset-discord.txt ничего не срабатывает
@Flowseal  есть ли предположение почему тогда работало и почему перестало сейчас?

**Reply by Flowseal** [reply]
Не могу подсказать, у меня рутрекер всегда работал и рабоатет

**Reply by lliinnii** [reply]
Магия какая-то я уже просмотрела всю сборку сравнила с текущей версией и кажется что нет ничего такого, что должно было анлокать rutracker да и в целом как-то иначе не должно было работать -___-

**Reply by lliinnii** [reply]
Хотя в случае с новой версией пишет 
we have 4 user defined desync profile(s) and default low priority profile 0
В случае старой
we have 5 user defined desync profile(s) and default low priority profile 0

Маловероятно, но может 1 из профилей что выпилился и содержал что-то связанное?

**Reply by lliinnii** [reply]
Все отставить тряску, я тупанула.
помимо 
rutracker.org
rutracker.com

Нужно еще захерачить 
*.rutracker.cc
static.rutracker.cc

**Reply by Beast968** [reply]
> Все, хватит трястись, я тупанула. помимо rutracker.org rutracker.com
> 
> Нужно еще захерачить *.rutracker.cc static.rutracker.cc

а *.rutracker.cc так точно будет работать?

**Reply by lliinnii** [reply]
> > Все, хватило трястись, я тупанула. кроме rutracker.org rutracker.com
> > Нужно еще захерачить *.rutracker.cc static.rutracker.cc
> 
> а *.rutracker.cc так точно будет работать?


Да, у меня все работает, вот мой файл list-general.txt (там правда еще разные сайты добавлены (не относящиеся к rutracker), но сам rutracker работает. 

На всякий случай уточню, что если вы вносите изменения файл когда запущена программа - ее нужно закрывать и запускать заново.  
[list-general.txt](https://github.com/user-attachments/files/18139451/list-general.txt)

**Comment by boroosku** [comment]
Добавил Spotify в list-general.txt, теперь работает веб-версия:

spotify.com
open.spotify.com
accounts.spotify.com

Но десктопное приложение не работает, выдает ошибку при входе на аккаунт:
![image](https://github.com/user-attachments/assets/1416e3e2-bd23-473c-9ab0-286b215dc8ba)

Можно ли это исправить?

**Reply by Eptiy** [reply]
Споти не будет так работать. Они прекратили деятельность в РФ и сами заблокировали нам доступ к сервису. Ты никак не можешь прослушивать треки с ру региона. Если у тебя в профиле споти стоит другой регион, то тебе нужно входить из-под впн этого региона раз в 14 дней (вроде), эти 14 дней он разрешает слушать из любого региона, потом выбрасывает из аккаунта.
Если купить премиум, то выбрасывать не будет, можно неограниченно слушать из РФ при любом выбранном в аккаунте регионе.

**Reply by Eptiy** [reply]
Почему у тебя веб заработал, не могу знать. Запрет обходит алгоритмы dpi, которые работают на ру провайдерах, когда РФ пригрозила споти блокировкой, те просто свернули шатёр и теперь сами ограничивают доступ из РФ.

**Reply by boroosku** [reply]
Спасибо за ответ

**Reply by elena-kanakova** [reply]
Думаю, что теперь со Spotify вопрос актуален. Сама компания все еще дает слушать premium подписки других стран на территории РФ (то есть не блокает наш IP), но РКН, походу, блокнул Spotify. Или Spotufy попал под раздачу блокировок CF, хрен его знает. Но проблема в том, что все работает, разве что песни не грузятся. История схожая с блокировками серверов CF - запрос туда идет, что-то начинает грузить, но обрывается. Ну и картинки не грузит тоже. Вопрос: может ли как-то запрет с этим помочь? По сути тут стратегия должна быть как с дискордом, ведь блокировка идет от провайдеров, а не самой компании. Хотя может они и замедлили его как ютуб... потому что крутится "идет загрузка" на треках и периодически что-то да пытается загрузится там.

**Comment by Eptiy** [comment]
@Flowseal мб добавишь лист от 1andrevich? Много кому надо больше ресурсов разблочить, но мало кто знает об этом листе.

**Comment by onroe000** [comment]
не получается зайти на soundcloud, добавил в list-general домен soundcloud.com, не сработало. может, нужны ещё какие-то домены, кто знает? нигде в инете не могу найти информацию по этому поводу

@Flowseal
![image](https://github.com/user-attachments/assets/ef247fc6-c72a-49d4-ac6e-003ef30bd39b)

**Reply by onroe000** [reply]
странно, но всё заработало. ничего не делал 0-0

**Reply by Eptiy** [reply]
Всякое бывает. У меня саундклауд иногда пролагивает и не прогружается.

Если что, найти большинство доменов популярных ресурсов можно [здесь](https://github.com/v2fly/domain-list-community/tree/master/data).

У саундклауда всего 2 есть по этим данным:
soundcloud.com
sndcdn.com

**Reply by onroe000** [reply]
> У саундклауда всего 2 есть по этим данным: soundcloud.com sndcdn.com

нашёл ещё один: soundcloud.app.goo.gl

а вообще, сайт не работал пока я не вынес домены ск на самый верх txt файла, вот так вот)

**Comment by Brescu** [comment]
предлагаю добавить ChatGPT для windows, майки релизнули вчера. С ру айпишников недоступно

**Reply by Eptiy** [reply]
Zapret не меняет твой ip, он только обходит блокировки, максируя трафик от них.

**Comment by fed210** [comment]
А где этот файл располагается? Имеется ввиду list-general.txt

**Reply by ekungurov** [reply]
Ты файлы искать не умеешь? Поставь FAR или Total Commander, хотя виндой тоже можно. 
Кроме того, можно скачать архив и найти файл там. В архиве файлов немного.

**Reply by elena-kanakova** [reply]
Интересно почему этот ответ упал мне на почту, как будто я задавала вопрос, хотя эту ветку вижу впервые)

**Reply by Dronatar** [reply]
> Интересно почему этот ответ упал мне на почту, как будто я задавала вопрос, хотя эту ветку вижу впервые)

https://github.com/Flowseal/zapret-discord-youtube/discussions/285#discussioncomment-13736597
